src/pytorch_lightning/CHANGELOG.md

```diff
@@ -55,6 +55,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed MPS device being unrecognized ([#13992](https://github.com/Lightning-AI/lightning/pull/13992))
 
 
+- Fixed `precision="mixed"` with `DeepSpeedStrategy` sharded context ([#14041](https://github.com/Lightning-AI/lightning/pull/14041))
```

Comment:
I'm not convinced that this is fixing anything since passing `mixed` is still mostly undefined. IMO the true fix is to do the proposal in https://github.com/Lightning-AI/lightning/issues/9956

Reply:
yes! I noticed it.. forgot to add the following in the description:

'mixed' isn't just configured with sharded context. Otherwise, it works normally with deepspeed.

Alternatively, we can remove `mixed`support completely from Deepspeed since this is true precision.

wdyt?

Reply:
> Alternatively, we can remove mixedsupport completely from Deepspeed since this is true precision.

I think this makes the most sense. And in line with #9956

Reply:
done!